topi/python/topi/cuda/nms.py

```diff
@@ -169,6 +168,9 @@ def calculate_overlap(out_tensor, box_a_idx, box_b_idx):
                                                         base_idx + offset_i + 2)
                                 with ib.if_scope(iou >= nms_threshold):
                                     p_out[base_idx + offset_i] = -1.0
+                ib.emit(tvm.make.Call(None, 'tvm_storage_sync',
```

Comment:
this makes sense to me, could you also add this to proposal op?

Reply:
Sure, I’ll do that.